### ISSUE TITLE
Nextcloud 27 upgrade breaks Nextcloud Office

### ISSUE THREAD

**Issue by karlemilnikka** [nextcloud/all-in-one#2773]
### Steps to reproduce
1. Upgrade to Nextcloud 27 by following the official instructions. 
2. Try opening a Nextcloud Office document or create a new one. 

### Expected behavior 
I’ve run into a very strange Nextcloud Office issue after upgrading to Nextcloud 27 (Nextcloud AIO 6.1.1). Downgrading solves the issue, but I thought I should mention it in case others run into the same issue. 

After upgrading, I can no longer open documents in Nextcloud Office. If I create a new document, I get an error message stating that “document loading failed”. If I try to open an existing document, I get a partially loaded Nextcloud Office interface with a document stating “cannot decrypt this file, probably this is a shared file. Please ask the file owner to reshare the file with you”. 

This issue is however not affecting templates. I can open templates without any issues. Everything seems to be in order in the settings app as well (settings/admin/richdocuments). Allowing all IP addresses in the WOPI requests field doesn’t solve the issue. 

### Host OS 
Ubuntu 20.04

#### Nextcloud AIO version 
Nextcloud AIO 6.1.1

#### Current channel 
Stable

**Comment by szaimen** [comment]
cc @juliushaertl do you know if something has changed on collabora side in code:23 that needs changes on AIO side?

**Comment by szaimen** [comment]
@karlemilnikka do you have something related in collabora or nextcloud logs?

**Comment by juliushaertl** [comment]
I don't think we had a related change in richdocuments but this sounds like https://github.com/nextcloud/richdocuments/issues/2996

**Comment by szaimen** [comment]
Thanks Julius! :)

Duplicate of https://github.com/nextcloud/richdocuments/issues/2996 then.
